src/modules/land_detector/FixedwingLandDetector.cpp

```diff
@@ -114,6 +114,14 @@ bool FixedwingLandDetector::_get_landed_state()
 		const float acc_hor = matrix::Vector2f(_acceleration).norm();
 		_xy_accel_filtered = _xy_accel_filtered * 0.8f + acc_hor * 0.18f;
 
+		// Check for angular velocity (disregard yaw)
+		float max_rotation_threshold = math::radians(_param_lndfw_rot_max.get());
+		val = _angular_velocity.xy().norm(); // TODO: check if this should be low pass filtered
```

Comment:
in MC landing detector the angular velocity is not filtered. Unsure if it might be necessary for FW - I don't know how noisy the data usually is in comparison.